src/pages/lists/hooks/useFilters/useFilters.ts

```diff
@@ -16,12 +16,29 @@ const useURLFilters = () => {
 
   const { getItem, setItem } = useSessionStorage(`${namespace}/filters`)
 
+  const sessionFilters = getItem() as string;
+
+  const setValues = (filters: string[]) => {
+    searchParams.set(FILTERS_URL_KEY, filters.join(','))
+
+    history.push(`${history.location.pathname}?${searchParams.toString()}`)
+
+    setItem(filters.join(','))
+  };
+
   useEffect(() => {
     const filtersURL = (searchParams.get(FILTERS_URL_KEY)?.split(',') || []).filter((filter) => {
       return !!filter
     }).join(',');
 
-    const sessionFilters = getItem() as string;
+    if(sessionFilters === filtersURL) {
+      return
```

Comment:
missed ";"

Reply:
Tiny nit on the formatting (missing space) around the `if`, too